### PR TITLE
 TestSong99BottlesAsTest

### PR DESCRIPTION
**Pull request by AselAkhmatova**
added test Song99BottlesAsTest

### CODE REVIEW

File: src/test/java/qa_java_beginners/Song99BottlesAsTest.java [hunk 1]
```diff
@@ -133,4 +138,39 @@ public void testSongLyricTest() {
...
+        for (int i = 0; i < pText.length; i++) {
+
+            int index = i + 1;
+
+            pText[i] = getDriver().findElement(By.xpath(
+                    "//div[@id='main']/p[" + index + "]")).getText();
+        }
+
+        String actualyResalt = "";
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesAsTest.java:166]: Delete empty line.

File: src/test/java/qa_java_beginners/Song99BottlesAsTest.java [hunk 2]
```diff
@@ -133,4 +138,39 @@ public void testSongLyricTest() {
...
+    @Test
+    public void testSearchJ() {
+
+        String expectedResalt = "All languages starting with the letter J are shown, sorted by Language.";
+
+        getDriver().get(URL);
+
+        getDriver().findElement(
+                By.xpath("//div/ul/li/a[@href='/abc.html']")).click();
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesAsTest.java:151]: You might delete empty line here.

File: src/test/java/qa_java_beginners/Song99BottlesAsTest.java [hunk 3]
```diff
@@ -133,4 +138,39 @@ public void testSongLyricTest() {
...
+        getDriver().findElement(
+                By.xpath("//div/ul/li/a[@href='j.html']")).click();
+
+        String[] pText = new String[2];
+
+        for (int i = 0; i < pText.length; i++) {
+
```
Comment (anonymoustuna) [inline src/test/java/qa_java_beginners/Song99BottlesAsTest.java:158]: You can delete this empty line.

**Review by anonymoustuna — APPROVED** [review]

File: src/test/java/qa_java_beginners/Song99BottlesAsTest.java (hunk 3 above)
Comment (AselAkhmatova) [reply src/test/java/qa_java_beginners/Song99BottlesAsTest.java:158]: ok, thanks

**Review by dmswlpk — APPROVED** [review]

**Pull request merged**